### PR TITLE
Update version number to 0.0.2

### PR DESCRIPTION
**Pull request by tshoang**
Update version number due to incorrect build in 0.0.1

### CODE REVIEW

**Review by cfsnook — COMMENTED** [review]
looks ok but why? I can't see any change other than the version number?
is the reason for bumping the service number because the service number increased!

File: ac.soton.eventb.emf.containment.releng/pom.xml [hunk 1]
```diff
@@ -18,45 +18,38 @@
   <!-- ID of this artifact -->
   <artifactId>releng</artifactId>
   <!-- REQUIRED: Version number (using semantic versioning) -->
-  <version>0.0.1.release</version>
+  <version>0.0.3.release</version>
```
Comment (cfsnook) [inline ac.soton.eventb.emf.containment.releng/pom.xml:21]: do we need to skip 0.0.2? (I don't think the version 0.0.2 was ever merged so is still part of this pull request?)

File: pom.xml [hunk 2]
```diff
@@ -21,15 +21,15 @@
...
+  <version>0.0.2.release</version>
   <packaging>pom</packaging>
   <parent>
     <!-- REQUIRED: Group ID is the feature ID -->
     <groupId>ac.soton.eventb.emf.containment</groupId>
     <!-- Name of the release engineering artifact -->
     <artifactId>releng</artifactId>
     <!-- REQUIRED: Version number of the release engineering artifact -->
-    <version>0.0.1.release</version>
+    <version>0.0.3.release</version>
```
Comment (cfsnook) [inline pom.xml:32]: do we need to skip 0.0.2? (I don't think the version 0.0.2 was ever merged so is still part of this pull request?)

Comment (tshoang) [reply pom.xml:32]: I think we can go directly to 0.0.3 now. 0.0.2 actually is the same as 0.0.1. Due to an incorrect build of 0.0.1, I have to increase it to 0.0.2.

Comment (cfsnook) [reply pom.xml:32]: Ok

**Review by cfsnook — APPROVED** [review]
looks good then

**Pull request merged**